pkgs/by-name/li/linux-wallpaperengine/package.nix

```
@@ -113,7 +112,7 @@ let
       runHook preInstall
 
       cp -r . $out
```

```suggestion
      cp --recursive --no-preserve=mode . $out
```

This should obsolete the chmod.